templates/partials/contacts_content.html

```diff
@@ -72,30 +72,25 @@ <h3 translate>{{card.label}}</h3>
         <div ng-show="selected.doc.type === 'clinic'" mm-auth="can_view_tasks" class="col-xs-2 tasks-title" translate>Tasks</div>
         <div class="card action-list children persons">
           <div class="accent"></div>
-          <div class="row" ng-repeat="child in selected.children.persons">
-            <a ui-sref="contacts.detail({ id: child.doc._id })" class=" item-summary cell col-xs-12">
-              <div ng-show="selected.doc.type !== 'clinic'" class="col-xs-12 detail">
-                <div class="cell-label">
-                  <i class="fa fa-star primary" ng-show="child.isPrimaryContact" translate translate-attr-title="Primary contact"></i>
-                  <span class="name">{{child.doc.name}}</span>
-                </div>
-                <div class="col-sm-3" ng-bind-html="child.doc.title"></div>
-              </div>
-
-              <div ng-show="selected.doc.type === 'clinic'" class="col-xs-11 detail">
-                <div class="cell-label">
-                  <i class="fa fa-star primary" ng-show="child.isPrimaryContact" translate translate-attr-title="clinic.field.contact"></i>
-                  <span class="name">{{child.doc.name}}</span>
-                </div>
-                <div>
-                  <span ng-bind-html="child.doc.date_of_birth | age"></span>
-                </div>
-              </div>
-              <div ng-show="selected.doc.type === 'clinic' && child.taskCount > 0" mm-auth="can_view_tasks" class="col-xs-1 tasks-count">
-                <span class="badge">{{child.taskCount}}</span>
-              </div>
-            </a>
-          </div>
+          <ul ng-if="selected.doc.type !== 'clinic'">
+            <mm-content-row ng-repeat="child in selected.children.persons"
+              id="child.doc._id"
+              route="'contacts'"
+              heading="child.doc.name"
+              summary="child.doc.title"
+              primary-contact="child.isPrimaryContact"
+            />
+          </ul>
+          <ul ng-if="selected.doc.type === 'clinic'">
+            <mm-content-row ng-repeat="child in selected.children.persons"
+              id="child.doc._id"
+              route="'contacts'"
+              heading="child.doc.name"
+              dob="child.doc.date_of_birth"
+              primary-contact="child.isPrimaryContact"
+              task-count="child.taskCount"
+            />
```

Comment:
Is there a way of combining these two things so that the `if`ing happens on `dob` and `task-count`?

Reply:
Yes you could have a ternary in the attribute value. You'd also have to add a ternary to `summary` which is only in the `!clinic` row. I gave it a try and it looks quite messy, so I've left it. If you feel strongly about it let me know and we can compare pros and cons  :)